.github/workflows/pre-release.yml

```
@@ -21,5 +21,8 @@ jobs:
 
       - name: Deploy to testpypi.org
         run: |
+          LATEST_RELEASE=${curl "https://api.github.com/repos/farridav/django-jazzmin/tags" | jq -r '.[0].name'/#v}
+          poetry version $LATEST_RELEASE
+          poetry version patch
```

Why patch? Should we release a prepatch before?
```suggestion
          poetry version prepatch
```